bears/java/CheckstyleBear.py

```diff
@@ -17,10 +17,14 @@
                                         ' not found.')
 class CheckstyleBear:
     """
-    Checks the code with ``checkstyle`` using the Google codestyle
-    specification.
+    Check Java code for possible style, semantic and design issues.
+
+    For more information, consult
+    <http://checkstyle.sourceforge.net/checks.html>.
     """
 
+    LANGUAGES = ("Java",)
```

Comment:
`("something")` is the same as `"something"` but we want it to be a tuple, standard notation is `("something",)`